lib/generators/react_on_rails/templates/base/base/config/initializers/react_on_rails.rb.tt

```diff
@@ -60,6 +60,12 @@ ReactOnRails.configure do |config|
   config.server_renderer_timeout = 20 # seconds
 
   ################################################################################
+  # I18N OPTIONS
+  ################################################################################
+  # Replace the following line to the location where you keep translation.js & default.js.
+  config.i18n_dir = Rails.root.join("client", "app", "libs", "i18n")
```

Comment:
@JasonYCHuang We added this here, with an invalid directory, and later we abort deploying or testing when this directory does not exist.